### DISCUSSION TITLE
How do you usually find internships outside of the repos?

### DISCUSSION THREAD

**Discussion by ShevinuM** [jenndryden/Canadian-Tech-Internships-and-New-Grad-2025#156] [General]
I'm not seeing positions for Canada a lot on LinkedIn. It's always the same bunch of companies I've applied to. Not sure if I'm doing anything wrong or if there actually aren't any positions

**Comment by kanjurer** [comment]
There actually aren't many positions :(
Market sucks 😭

**Reply by ShevinuM** [reply]
This is sad 😭

**Reply by kanjurer** [reply]
Issok :") We together
